host/src/FunctionsNetHost/Logger.cs

```diff
@@ -8,22 +8,13 @@ namespace FunctionsNetHost
     internal static class Logger
     {
         private static readonly string LogPrefix;
+        private static readonly bool IsTraceLogEnabled;
 
         static Logger()
         {
-#if !DEBUG
-            LogPrefix = "LanguageWorkerConsoleLog";
-#else
-            LogPrefix = "";
-#endif
-        }
-
-        internal static bool IsTraceLogEnabled
-        {
-            get
-            {
-                return string.Equals(EnvironmentUtils.GetValue(EnvironmentVariables.FunctionsNetHostTrace), "1");
-            }
+            IsTraceLogEnabled = string.Equals(EnvironmentUtils.GetValue(EnvironmentVariables.EnableTraceLogs), "1");
```

Comment:
Had a chat with @kshyju. It would be helpful to re-evaluate this at specialization time so the site has the ability to override this behavior.

Reply:
@fabiocav Pushed an update which reloads the logger configuration on specialization.

Reply:
thank you!